runtime/default.policy

```diff
@@ -96,7 +170,7 @@ rules:
     description: unauthorized file accessing access logs
     expression: >-
       open.filename in ["/run/utmp", "/var/run/utmp", "/var/log/wtmp"] &&
-      process.filename not in ["/usr/bin/login", "/usr/sbin/sshd", "/usr/bin/last", "/usr/bin/who", "/usr/bin/w", "/usr/bin/sudo", "/usr/sbin/cron"]
+      process.filename not in ["/usr/bin/login", "/usr/sbin/sshd", "/usr/bin/last", "/usr/bin/who", "/usr/bin/w", "/usr/bin/sudo", "/usr/sbin/cron", "/usr/sbin/VBoxService"]
```

Comment:
I'm not sure we should put this one. `VBoxService` is related to you vagrant VM and would probably not apply in production 😅

Reply:
@Gui774ume although it probably would not apply in production it's noise anyways correct? Why shouldn't we tune out the noise wherever possible?

Reply:
We don't know if / how a user might use Virtualbox. I feel like this is an infrastructure specific decision that we try to fit into a generic policy. If we add VBoxService, why not all the other virtualization services out there ?

Reply:
(also, I agree about noise but I believe that running after all possible process that might touch a file for a legitimate purpose is not a sustainable way to enrich the default policy)

Reply:
But feel free to ignore, this is just my 2cents 😃